1-python-wrapper/requirements.txt

```diff
@@ -0,0 +1,3 @@
+pandas==1.1.3
+scikit-learn>0.23
+flask
```

Comment:
Everything in this file should be pinned if we don't want this to break at some point. Is it too early to add `pip-compile` to the workflow?

Reply:
I am not familiar with pip-compile. I can add setup.py and add these requirements there, is that equivalent? I will add a flask version :)